diff --git a/home/khaes/phd/courses/wasp-4/tool/tmp/Patch-Explainer-Test/src/main/java/NumberAnalyzer.java b/tmp/src3149222994637392973/NumberAnalyzer.java
index 6769336..9e96e7a 100644
--- a/home/khaes/phd/courses/wasp-4/tool/tmp/Patch-Explainer-Test/src/main/java/NumberAnalyzer.java
+++ b/tmp/src3149222994637392973/NumberAnalyzer.java
@@ -1,10 +1,11 @@
 public class NumberAnalyzer {
-    public static String analyze(int x){
+    public static java.lang.String analyze(int x) {
         if (x > 0)
-            return x + " is smaller than zero.";
+            return x + "is not smaller";
         else if (x == 0)
             return x + " is exactly zero.";
         else
             return x + " is greater than zero.";
+
     }
-}
+}
